### ISSUE TITLE
MovementDescriber should take an optional ModelViewTransform2

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/scenery-phet#547]
Needed for https://github.com/phetsims/molecules-and-light/issues/201, but also after some conversation on a11y-dev slack channel. MovementDescriber is responsible for generating descriptions and alerts about directional  movement. It is driven by a provided locationProperty which is in the model coordinate frame. But we want to describe what the movement looks like in the view coordinate frame. And so MovementDescriber should take an optional modelViewTransform that transforms the points used to describe motion.

**Comment by jessegreenberg** [comment]
The above is pretty much all that is needed. Ill test it out in molecules-and-light before requesting review.

**Comment by zepumph** [comment]
Looks great to me, thanks for doing this!

**Comment by jessegreenberg** [comment]
Thanks for review
